src/components/progress/progress.scss

```diff
@@ -0,0 +1,117 @@
+@import "../../themes/ionic.globals";
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:              rgb(16,108,200) !default;
+
+/// @prop - The background color of the progress bar
+$progress-buffer-color:       rgb(170,209,249) !default;
+
+/// @prop - The animation duration of the progress bar
+$progress-animation-duration:   2s !default;
+
+/// @prop - The speed at which the progress values transition, this is useful for determinate and buffer modes
+$progress-values-transition:    0.3s !default;
+
+/// @prop - The speed at which the buffer circles will be animating
+$progress-buffer-circle-speed:  0.3s !default;
+
+/// @prop - SVG to use for the buffering circles whenever the progress bar is buffering
+$progress-buffer-circles-svg:   "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIj8+Cjxzdmcgd2lkdGg9IjEyIiBoZWlnaHQ9IjQiIHZpZXdQb3J0PSIwIDAgMTIgNCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgogIDxlbGxpcHNlIGN4PSIyIiBjeT0iMiIgcng9IjIiIHJ5PSIyIj4KICAgIDxhbmltYXRlIGF0dHJpYnV0ZU5hbWU9ImN4IiBmcm9tPSIyIiB0bz0iLTEwIiBkdXI9IjAuNnMiIHJlcGVhdENvdW50PSJpbmRlZmluaXRlIiAvPgogIDwvZWxsaXBzZT4KICA8ZWxsaXBzZSBjeD0iMTQiIGN5PSIyIiByeD0iMiIgcnk9IjIiIGNsYXNzPSJsb2FkZXIiPgogICAgPGFuaW1hdGUgYXR0cmlidXRlTmFtZT0iY3giIGZyb209IjE0IiB0bz0iMiIgZHVyPSIwLjZzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgLz4KICA8L2VsbGlwc2U+Cjwvc3ZnPgo=";
+
+ion-progress {
+  height: 5px;
+  width: 100%;
+  position: relative;
+  overflow: hidden;
+  display: block;
+  @keyframes indeterminate {
+    0% {
+      width: 0%;
+
+      left: 0%;
```

Comment:
I don't use vs-code, so can't help much, but this seems like what you need - https://marketplace.visualstudio.com/items?itemName=adamwalzer.scss-lint